game/igdb_wrapper.py

```diff
@@ -0,0 +1,108 @@
+import os
+import requests
+from game.igdb_data_format import format_data
+
+
+class IGDBWrapper:
+    db_requests_url = 'https://api.igdb.com/v4'
+    access_token_url = 'https://id.twitch.tv/oauth2/token'
+
+    def __init__(self) -> None:
+        self.client_id = self.get_client_id()
+        self.client_secret = self.get_client_secret()
+        self.grant_type = self.get_grant_type()
+
+    @staticmethod
+    def get_client_secret() -> str:
+        """Gets client secret from the project environment"""
+
+        return os.getenv('IGDB_CLIENT_SECRET')
```

Comment:
could you please create 2 files: `.env` and `.env.example`

in `.env` there will be actual values: 
```
IGDB_CLIENT_ID=123
IGDB_CLIENT_SECRET=456
```
And in .env.example there will be only
```
IGDB_CLIENT_ID=
IGDB_CLIENT_SECRET=
```

Then in readme it's usually said "please copy .env.example content into .env and fill credentials"

(same for twitter)

Reply:
Then you'll be able just to use `soure .env` before running application to import all needed variables